scripts/release-bit-verification.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/sh
+
+############################################################################
+#  PREREQUISITES FOR THIS SCRIPT
+# 1. Redistributable-binary(.rpm) should be passed as the first argument
+# 2. Login to the cluster should be done prior to running this script
+# 3. The cluster should be in a state where it can be used for testing
+#
+# USAGE:
+# ./check-binary.sh redistributable-binary
+#
+# Example: ./check-binary.sh ~/Downloads/odo-redistributable-2.4.3-1.el8.x86_64.rpm
+#
+
+shout() {
+  echo "--------------------------------------------------------------------------"
+}
+# Check SHASUM for all the binary files and there should be no difference
+
+# Extract from rpm file 
+rpm2cpio ${1} | cpio -idmvD /tmp
```

Comment:
Could you create a temp directory inside tmp and work on it, instead of working directly into /tmp?

Reply:
Sure! But is there any specific reasoning behind that?

Reply:
/tmp is used by other scripts, it is better to isolate your files in a single directory, instead of working on some *common* subdirectories like /tmp/usr, etc.

Reply:
Understood! Thanks, @feloy 😃